### PR TITLE
feat: add discord labels to metrics

### PR DESCRIPTION
**Pull request by jlehtimaki**
This PR adds discord IDs to the metrics.
Also removes from one struct dependency from `gorelayer` to use `networks` repo schema instead.

### CODE REVIEW

File: pkg/collector/collector.go [hunk 1]
```diff
@@ -192,3 +216,12 @@ func (wb WalletBalanceCollector) Collect(ch chan<- prometheus.Metric) {
 
 	log.Debug("Stop collecting", zap.String("metric", walletBalanceMetricName))
 }
+
+func getDiscordIDs(ops []config.Operator) string {
+	var ids []string
+	for _, op := range ops {
+		ids = append(ids, op.Discord.ID)
```
Comment (kayano) [inline pkg/collector/collector.go:223]: I think we should append `op.Discord.ID` to `ids` only if it is not empty string.

Comment (jlehtimaki) [reply pkg/collector/collector.go]: Added a check and test cases for this

File: pkg/collector/collector_test.go [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+			expected: "",
+		},
+		{
+			name:     "Empty Input",
+			ops:      []config.Operator{{}, {}, {}},
+			expected: "",
+		},
+	}
+	for _, tc := range testCases {
+		res := getDiscordIDs(tc.ops)
```
Comment (kayano) [inline pkg/collector/collector_test.go:59]: I recommend to wrap it in `t.Run` like
```
	for _, tc := range testCases {
		t.Run(tc.name, func(t *testing.T) {
			res := getDiscordIDs(tc.ops)
			assert.Equal(t, tc.expected, res)
		})
	}
```
This way it is easy to spot which test case failed and a test case name is being used. To show it I've made tests failing like
```
=== RUN   TestDiscordIDs
    collector_test.go:60:
        	Error Trace:	/Users/tnowak/code/archway-network/relayer_exporter/pkg/collector/collector_test.go:60
        	Error:      	Not equal:
        	            	expected: "123456,12312387"
        	            	actual  : "12312387"

        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1 @@
        	            	-123456,12312387
        	            	+12312387
        	Test:       	TestDiscordIDs
    collector_test.go:60:
        	Error Trace:	/Users/tnowak/code/archway-network/relayer_exporter/pkg/collector/collector_test.go:60
        	Error:      	Not equal:
        	            	expected: ","
        	            	actual  : ""

        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1 @@
        	            	-,
        	            	+
        	Test:       	TestDiscordIDs
--- FAIL: TestDiscordIDs (0.00s)
```
vs
```
=== RUN   TestDiscordIDs
=== RUN   TestDiscordIDs/All_Valid_IDs
    collector_test.go:61:
        	Error Trace:	/Users/tnowak/code/archway-network/relayer_exporter/pkg/collector/collector_test.go:61
        	Error:      	Not equal:
        	            	expected: "123456,12312387"
        	            	actual  : "12312387"

        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1 @@
        	            	-123456,12312387
        	            	+12312387
        	Test:       	TestDiscordIDs/All_Valid_IDs
=== RUN   TestDiscordIDs/Some_Invalid_IDs
=== RUN   TestDiscordIDs/No_Valid_IDs
=== RUN   TestDiscordIDs/Empty_Input
    collector_test.go:61:
        	Error Trace:	/Users/tnowak/code/archway-network/relayer_exporter/pkg/collector/collector_test.go:61
        	Error:      	Not equal:
        	            	expected: ","
        	            	actual  : ""

        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1 @@
        	            	-,
        	            	+
        	Test:       	TestDiscordIDs/Empty_Input
--- FAIL: TestDiscordIDs (0.00s)
```
I think in the second report it is easier to understand which test case failed.

Comment (jlehtimaki) [reply pkg/collector/collector_test.go]: Yes, makes sense!

**Review by kayano — APPROVED** [review]

**Pull request merged**